### DISCUSSION TITLE
Welcome to website Discussions!

### DISCUSSION THREAD

**Discussion by sdottaka** [WinMerge/website#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lmstearn** [comment]
Just popped in to say hi, wonderful to see Winmerge at Github after the many productive years at Sourceforge. I guess they never wanted to see you go so soon, but hey, at least they are happy to host older versions of Winmerge that work for previous editions of Windows. Had a little look around here, seems like you have settled in the new home rather well.
Look forward to perusing the [cafepress shop](https://cafepress.com/winmerge) when its back up, if there is coffee on the shelves, I'm in!  Plenty of coffee and tea back here at github, [lots of tea](https://github.com/rkazakov/tea-club/blob/master/COLLECTION.md) in fact. Although passersby will not encounter a saloon or bar setup in [Discussions](https://github.com/community/community/discussions) to date, they are a pretty friendly lot all the same.
And thanks for developing Winmerge,  just installed 2.16.24, on W11, looks fantastic, thanks. :thumbsup:
